randomGraphs.py

```diff
@@ -0,0 +1,42 @@
+from random import randint
+from random import sample
+
+VERTICES_MIN = 20
+VERTICES_MAX = 60
+
+EDGES_MIN = 1 #Per Vertex
+EDGES_MAX = 4 #Per Vertex
```

Comment:
Weren't we going to do up to 6 edges per vertex max?

Reply:
Yeah, the problem here was that I randomly picked vertices to make the edges but all vertices might pick the same vertex. I'm not sure how to fix this problem.

Another problem here is that a vertex might have a self-loop. I will fix this now.

Reply:
This should be fixed with the new commit.